common/create_db_cmd

```diff
@@ -0,0 +1,2 @@
+  o Add a sanitized command executor for database creation and re-enable
+  user database creation on CouchServerState via command line.
```

Comment:
I saved this on the wrong place.
Easy to move on rebase, but rebase is not that easy because of latest change.